data/packages/expl3.json

```diff
@@ -4750,9 +4816,15 @@
     {
       "name": "int_gincr:c"
     },
+    {
+      "name": "int_gset:NV"
+    },
```

Comment:
It's a typo that the source file `l3int.dtx` at l3kernel 2024-11-02 used `\int_gset:MV` (should be N, not M).

The typo has already been fixed in latex3 dev (https://github.com/latex3/latex3/commit/20fa9ac27aa5439fa5f5f6cfc913ec67efcaeeba), and also manually corrected here.